src/components/Home/ConnectionCard/ConnectionCard.tsx

```diff
@@ -29,8 +31,8 @@ export const ConnectionCard: FC<ConnectionCardProps> = ({
 
   return (
     <div className="p-5 h-full">
-      <div className="bd-card transition-colors">
-        <div className="font-bold text-lg flex items-center">
+      <section className="bd-card transition-colors">
+        <article className="font-bold text-lg flex items-center">
```

Comment:
[I feel like if it's not real content you could skip the use of `section`/`article`](https://bitsofco.de/sectioning-content-in-html5/). In that's case a `div`/`span`/`p` is fine.